specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples/CancelJobExecution.json

```diff
@@ -5,7 +5,7 @@
     "serverName": "server1",
     "jobAgentName": "agent1",
     "jobName": "job1",
-    "jobExecutionId": "5555-6666-7777-8888-999999999999",
+    "jobExecutionId": "5A86BF65-43AC-F258-2524-9E92992F97CA",
```

Comment:
do we know why we change it?

Reply:
my purpose is to fix example issue for ring0 RP control plane swaggers under default tag
in this example, it report INVALID_FORMAT error about format uuid: 5555-6666-7777-8888-999999999999

Reply:
@ericshape , should we update examples here then you help apply these changes in the ADO repo coz we don't have auto-sync in place yet?